lib/common/resources/otel-collector-config-new.yml

```diff
@@ -0,0 +1,1622 @@
+#
```

Comment:
Remove this file, should not be checked in!

Reply:
done